README.md

```diff
@@ -164,9 +155,6 @@ Additional steps are required for the different output types.
   - enable Google Sheets API for this project
```

Comment:
Are we supporting google sheets? if not - this can be removed.

Reply:
I'll handle this as part of a cleanup I'm doing

Reply:
Handled under https://github.com/DEFRA/forms-runner/pull/163